### DISCUSSION TITLE
How to get popular movies list for 2021, 2022 etc?

### DISCUSSION THREAD

**Discussion by neochine** [cinemagoer/cinemagoer#493] [Q&A]
https://www.imdb.com/list/ls534750065/
Something like this
get_popular_movies(year=2022) 
get_popular_movies(year=2021)

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
